change/@fluentui-react-flex-45808014-2cfe-4167-ac15-d8887f399e8a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Is this supposed to be `patch` or `prerelease`?

Reply:
`@fluentui/react-flex` have not been bumped yet for a release, it also has following config for `beachball`:

```json
  "beachball": {
    "tag": "latest",
    "disallowedChangeTypes": [
      "major",
      "prerelease"
    ]
  }
```

So, yes, only `patch` is a possible option 🐱